src/core/utils/remote-feature-config.ts

```diff
@@ -65,7 +71,9 @@ export const isFeatureActive = (feature: RemoteFeature): boolean => {
     }
 
     if (
-        (feature === RemoteFeature.NEAR ||
+        (feature === RemoteFeature.BETA_BADGE ||
+            feature === RemoteFeature.ZIL ||
```

Comment:
Check my branch. - there is a thing to consider.
If in the list that comes from firebase you see * then the feature will be active.
This will help us show to all users specific users